### ISSUE TITLE
AWS Toolkit Property 'on' Error

### ISSUE THREAD

**Issue by ramongoncalves** [aws/aws-toolkit-vscode#701]
**Describe the bug**
On execute the command "AWS: Create new SAM application", a TypeError is happens and the property 'on' is undefined.

**To Reproduce**

**Expected behavior**
Open inputs for app wizard.

**Screenshots**
![image](https://user-images.githubusercontent.com/2387734/61489028-15504180-a980-11e9-9208-230e01ca6679.png)

**Desktop (please complete the following information):**
 - OS: Mac Os Mojave
 - Visual Studio Code Version: 1.36.1 (1.36.1)
 - AWS Toolkit for Visual Studio Code Version: 1.0.0

**Additional context**
``` 2019-07-18 16:51:15 [ERROR]: An error occurred while creating a new SAM Application. Check the logs for more information by running the "View AWS Toolkit Logs" command from the Command Palette.
2019-07-18 16:51:15 [ERROR]: TypeError: Cannot read property 'on' of undefined
	at ChildProcess.<anonymous> (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:60:38)
	at Generator.next (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:11:71)
	at new Promise (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:7:12)
	at ChildProcess.start (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:55:16)
	at ChildProcess.<anonymous> (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:34:28)
	at Generator.next (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:11:71)
	at new Promise (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:7:12)
	at run.Promise (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:30:59)
	at new Promise (<anonymous>)
	at ChildProcess.<anonymous> (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:30:26)
	at Generator.next (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:11:71)
	at new Promise (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:7:12)
	at ChildProcess.run (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/utilities/childProcess.js:29:16)
	at DefaultSamCliProcessInvoker.<anonymous> (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliInvoker.js:45:39)
	at Generator.next (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliInvoker.js:11:71)
	at new Promise (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliInvoker.js:7:12)
	at DefaultSamCliProcessInvoker.invoke (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliInvoker.js:42:16)
	at SamCliInfoInvocation.<anonymous> (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliInfo.js:24:59)
	at Generator.next (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliInfo.js:11:71)
	at new Promise (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliInfo.js:7:12)
	at SamCliInfoInvocation.execute (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliInfo.js:23:16)
	at DefaultSamCliValidatorContext.<anonymous> (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliValidator.js:128:37)
	at Generator.next (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliValidator.js:11:71)
	at new Promise (<anonymous>)
	at __awaiter (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliValidator.js:7:12)
	at DefaultSamCliValidatorContext.getSamCliInfo (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliValidator.js:126:16)
	at DefaultSamCliValidator.<anonymous> (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliValidator.js:70:65)
	at Generator.next (<anonymous>)
	at fulfilled (/Users/xxxxxx/.vscode/extensions/amazonwebservices.aws-toolkit-vscode-1.0.0/out/src/shared/sam/cli/samCliValidator.js:8:58)
```

**Comment by kiiadi** [comment]
@ramongoncalves what kind of application are you trying to create (e.g netcore, node, python etc)? At what point in the new project process does the error occur? Is it on the actual create (e.g. after you've supplied a folder and project name?) or is it on the first screen after you've run the "AWS: Create new SAM application" command?

**Comment by bryceitoc9** [comment]
@ramongoncalves Are you still having issues with this?

**Comment by mshirozian** [comment]
@ramongoncalves  Having the same issue.  Did you ever figure this out?

**Comment by mshirozian** [comment]
@kiiadi Get this message after you submit command.  "An error occurred while creating a new SAM Application. Check the logs for more information by running the "View AWS Toolkit Logs" command from the Command Palette."  Log looks like that screenshot attached to this issue

**Comment by justinmk3** [comment]
Likely fixed by https://github.com/aws/aws-toolkit-vscode/pull/978 , which is included in Toolkit 1.12 (released today).
